server/models/trips.js

```diff
@@ -0,0 +1,50 @@
+
+export default (sequelize, DataTypes) => {
+  const Trips = sequelize.define('trips', {
+    user_id: {
+      type:DataTypes.INTEGER,
+      allowNull: false,
+    },
+    line_manager_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    origin: {
+      type:DataTypes.STRING,
+      allowNull: false,
+    },
+    destination: {
+      type:DataTypes.STRING,
+      allowNull: false,
+    },
+    dateOfTravel: {
+      type:DataTypes.STRING,
+      allowNull: false,
+    },
+    dateOfReturn:{
+      type:DataTypes.STRING,
+    },
+    travelReason: {
+      type:DataTypes.STRING,
+      allowNull: false,
+    },
+    accommodation_id:{
+      type:DataTypes.INTEGER,
+      allowNull: false,
+    },
+    type:{
+      type:DataTypes.STRING,
+    },
+  }, {});
+
+  Trips.associate =(models)=> {
+    Trips.hasOne(models.triprequests, {
```

Comment:
@irfiacre It would be better to define this association as belongsTo. Because many trips might belong to on trip_request. For example, In the case of multi-cities travel request, there will be more than 1  trip and all of them shall belong to one trip-request.